src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -2,11 +2,32 @@
 <html>
 <head>
     <meta charset="utf-8" />
-    <meta http-equiv="refresh" content="10;url=https://docs.microsoft.com/cli/azure/">
+    <meta http-equiv="refresh" content="120;url=https://docs.microsoft.com/cli/azure/">
     <title>Login successfully</title>
+    <style>
+        body {
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
+        }
+
+        code {
+            font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+            display: inline-block;
+            background-color: rgb(242, 242, 242);
+            padding: 12px 16px;
+            margin: 8px 0px;
+        }
+    </style>
 </head>
 <body>
-    <h4>You have logged into Microsoft Azure!</h4>
-    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 10 seconds.</p>
+    <h3>You have logged into Microsoft Azure!</h3>
+    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 2 minutes.</p>
+    <h3>Announcements</h3>
+    <p>[Windows only] Starting in May 2023, Azure CLI will authenticate using the <a href="https://learn.microsoft.com/en-us/windows/uwp/security/web-account-manager">Web Account Manager</a> (WAM) broker by default.</p>
+    <p>To help us collect feedback on the new login experience, you may opt-in to use WAM by running the following command:</p>
+    <code>
+        az config set core.allow_broker=true<br>
+        az account clear<br>
+        az login
```

Comment:
@jiasli can we simplify this bloc of code?

Reply:
Out of curiosity, @jiasli, is the "az account clear" necessary? My understanding is, after enabling broker, the subsequent "az foo bar" may fail and then Azure CLI's preexisting error message will guide end user to do an "az login" anyway.

Reply:
> after enabling broker, the subsequent "az foo bar" may fail and then Azure CLI's preexisting error message will guide end user to do an "az login" anyway.

Relying on the error impairs user experience, right. 🙂

My consideration is that we want users to run `az account clear` so that the existing MSAL cache is purged, and make sure no credentials are left behind.